packages/@react-types/button/src/index.d.ts

```diff
@@ -39,10 +39,15 @@ export interface SpectrumButtonProps extends ButtonProps {
 }
 
 export interface SpectrumActionButtonProps extends ButtonProps {
+  /** An icon to display in the ActionButton */
   icon?: ReactElement,
+  /** Whether the ActionButton should be displayed with a [quiet style](https://spectrum.adobe.com/page/action-button/#Quiet) */
   isQuiet?: boolean,
+  /** Whether the ActionButton should be displayed with a [selected state](https://spectrum.adobe.com/page/action-button/#Selected) */
   isSelected?: boolean,
+  /** Whether the ActionButton should be displayed with a [emphasized style](https://spectrum.adobe.com/page/action-button/#Emphasis) */
   isEmphasized?: boolean,
+  /** Whether the ActionButton should be displayed with a [hold icon](https://spectrum.adobe.com/page/action-button/#Hold-icon) */
```

Comment:
Missing periods at the end of these descriptions.
I personally how you added the links to the spectrum docs in the prop types themselves, should we do this for the same types in other component docs? Open to discussion, we should decide as a group to stay consistent